server/src/main/java/org/spine3/server/command/CommandBus.java

```diff
@@ -124,27 +127,37 @@ public void unregister(CommandHandler handler) {
      *
      * <p>If {@code Command} instance is passed, its message is extracted and validated.
      *
-     * @param command the command message to check
+     * @param commandOrMessage the command message to check
      * @return the result of {@link Responses#ok()} if the command is supported,
      *         {@link CommandValidation#unsupportedCommand(Message)} otherwise
      */
-    public Response validate(Message command) {
-        checkNotNull(command);
-        final CommandClass commandClass = CommandClass.of(command);
+    public Response validate(Message commandOrMessage) {
```

Comment:
Let's assume the parameter is only a `commandMessage`.

See the code of the `BoundedContext` that checks validity of the `CommandContext`. It is also a part of command validation. This code may belong to `CommandBus`, if it should be entitled to check for namespaces in a multi-tenant application.